doc/src/cylc-user-guide/siterc.tex

```diff
@@ -748,6 +753,142 @@ \subsection{[hosts]}
+utilities such as \lstinline=cylc gscan=. Additionally configure host
+selection settings specifying how to determine the most suitable run host at
+any given time from those configured.
+
+\subsubsection[run hosts]{[suite servers] \textrightarrow run hosts }
+
+A list of allowed suite run hosts. One of these hosts will be appointed for
+a suite to start up on if an explicit host is not provided as an option to
+a \lstinline=run= or \lstinline=restart= command.
+
+\begin{myitemize}
+\item {\em type:} comma-separated list of host names and/or IP addresses.
+\item {\em default:} \lstinline=localhost=
+\end{myitemize}
+
+\subsubsection[scan hosts]{[suite servers] \textrightarrow scan hosts }
+
+A list of hosts to scan for running suites.
+
+\begin{myitemize}
+\item {\em type:} comma-separated list of host names and/or IP addresses.
+\item {\em default:} \lstinline=localhost=
+\end{myitemize}
+
+\subsubsection[run ports]{[suite servers] \textrightarrow run ports }
+
+A list of allowed ports for Cylc to use to run suites. Note that only one
+suite can run per port for a given host, so the length of this list
+determines the maximum number of suites that can run at once per suite host.
+This config item supersedes the deprecated settings \lstinline=base port=
+and \lstinline=maximum number of ports=, where the base port is equivalent to
+the first port, and the maximum number of ports to the length, of this list.
+
+\begin{myitemize}
+\item {\em type:} string in the format \lstinline=X .. Y= for
+ \lstinline@X <= Y@ where \lstinline=X= and \lstinline=Y= are integers.
+\item {\em default:} \lstinline=43001 .. 43100= (equivalent to the list
+\lstinline=43001, 43002, ... , 43099, 43100=)
+\end{myitemize}
+
+\subsubsection[scan ports]{[suite servers] \textrightarrow scan ports }
+
+A list of ports to scan for running suites on each host set in scan hosts.
+
+\begin{myitemize}
+\item {\em type:} string in the format \lstinline=X .. Y= for
+ \lstinline@X <= Y@ where \lstinline=X= and \lstinline=Y= are integers.
+\item {\em default:} \lstinline=43001 .. 43100= (equivalent to the list
+\lstinline=43001, 43002, ... , 43099, 43100=)
+\end{myitemize}
+
+\subsubsection[run host select]{[suite servers] \textrightarrow [[run host select]]}
+
+Configure thresholds for excluding insufficient hosts and a method for
+ranking the remaining hosts to be applied in selection of the most suitable
+\lstinline=run host=, from those configured, at start-up whenever a set host
+is not specified on the command line via the \lstinline@--host=@ option.
+
+\paragraph[rank]{[suite servers] \textrightarrow [[run host select]] \textrightarrow rank}
+
+The method to use to rank the \lstinline=run host= list in order of
+suitability.
+
+\begin{myitemize}
+\item {\em type:} string (which must be one of the options outlined below)
+\item {\em default:} \lstinline=random=
+\item {\em options:}
+    \begin{myitemize}
+    \item {\bf random} - rank for suitability and select a host at random
```

Comment:
I personally think the 'rank' setting subsection should only detail the ranking step, as the run host select setting paragraph details the whole process of exclusion then ranking, but if you think this would be especially clear, I will change it?

Reply:
I assumed that's what "rank for suitability" meant. What was it intended to mean?

Reply:
That is of course what it means. I'm a little confused, I thought you were suggesting to add in the text you commented for this quote? I am saying I do not think it would not really be appropriate for this section, but if you feel strongly I am happy to add it in regardless.

Reply:
It just wasn't clear to me what "rank for suitability" means, if we could clarify a little.